general_rules/ExternalDevices.tex

```diff
@@ -3,35 +3,34 @@
 % External Devices
 % 
 % %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\section{External devices}
+\section{External Devices}
 \label{rule:robot_external_devices}
-Everything which is not part of the robot is considered an \iterm{external device}.
+Everything not part of the robot is considered an \iterm{external device}.
```

Comment:
This one I know

Either 
- Everything not being
- Everything which is not

Reply:
I’d accept the change as is. “Everything not part of the robot...”